scene/gui/code_edit.h

```diff
@@ -52,6 +52,12 @@ class CodeEdit : public TextEdit {
 		KIND_PLAIN_TEXT,
 	};
 
+	enum CodeblockGuideStyle {
```

Comment:
The entire code mentioning it could be all shortened to **CodeblockGuideStyle** for consistency, but if it's always lines it'd be strange. There's also a bit of a question on whether it should be "_CodeBlock_" or "_Codeblock_" because I've seen it mentioned with spacing in between ("_Code Block_"), and even this PR's title calls it that way.